src/main/java/BasicMAPF/Solvers/LargeNeighborhoodSearch/LargeNeighborhoodSearch_Solver.java

```diff
@@ -87,15 +89,21 @@ public LargeNeighborhoodSearch_Solver(I_SolutionCostFunction solutionCostFunctio
         this.sharedSources = Objects.requireNonNullElse(sharedSources, false);
         this.reactionFactor = Objects.requireNonNullElse(reactionFactor, 0.01);
         this.neighborhoodSize = Objects.requireNonNullElse(neighborhoodSize, 5);
+        this.TransientMAPFGoalCondition = Objects.requireNonNullElse(TransientMAPFGoalCondition, false);
 
-        super.name = this.destroyHeuristics.size() > 1 ? "ALNS" : ("LNS-" + this.destroyHeuristics.get(0).getClass().getSimpleName());
+        if (this.TransientMAPFGoalCondition) {
+            super.name = "LNSt";
+        }
+        else {
+            super.name = this.destroyHeuristics.size() > 1 ? "LNS" : ("LNS-" + this.destroyHeuristics.get(0).getClass().getSimpleName());
+        }
```

Comment:
```suggestion
        super.name = (destroyHeuristics.size() > 1 ? "A" : "") + "LNS" + (TransientMAPFGoalCondition ? "t" : "") +  (destroyHeuristics.size() == 1 ? "-" + destroyHeuristics.get(0).getClass().getSimpleName()) : "");
```

Great :) but how about adding it to the existing instead of replacing?